README.md

```diff
@@ -1,2 +1,20 @@
 # Hello Microverse test
- test for fix com
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Description the project.
```

Comment:
- Please add a proper description of the project. maybe like `This is a Hello Microverse project using HTML,CSS, and git`
- Also please add your social links. You can reference [this template](https://github.com/microverseinc/readme-template) and edit it to suit your project.